raco/backends/sql/catalog.py

```diff
@@ -107,10 +107,12 @@ def _convert_zeroary_expr(self, cols, expr, input_scheme):
 
     def _convert_unary_expr(self, cols, expr, input_scheme):
         input = self._convert_expr(cols, expr.input, input_scheme)
-        if isinstance(expr, expression.MAX):
-            return func.max(input)
-        if isinstance(expr, expression.MIN):
-            return func.min(input)
+
+        fname = expr.__class__.__name__.lower()
```

Comment:
Good point. I was coming at this with the backwards assumption that if it isn't in SQLAlchemy then we shouldn't support it. The requirement should be the backend SQL database supports it. For this reason, we should factor out the functionality (ie I'd like PushIntoSQL to be useful for more than just Postgres)